README.md

```diff
@@ -1,449 +1,41 @@
 # Kinde Android SDK
 
-## Overview
-Integrate Kinde authentication with your Android app. Simply configure, register, log in, and log out, and the authentication state is securely stored across app restarts.
+The Kinde SDK for Android
 
-These instructions assume you already have a Kinde account. You can register for free [here](https://app.kinde.com/register) (no credit card required).
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://makeapullrequest.com) [![Kinde Docs](https://img.shields.io/badge/Kinde-Docs-eee?style=flat-square)](https://kinde.com/docs/developer-tools) [![Kinde Community](https://img.shields.io/badge/Kinde-Community-eee?style=flat-square)](https://thekindecommunity.slack.com)
 
-## Install
-KindeSDK is available through [Maven](https://search.maven.org/). To install it, simply add the following line to your build.gradle:
+## Development
```

Comment:
@muhammadatif-devsinc Please remove the `Development` section. Since we want all development to be done against the corresponding generator: https://github.com/kinde-oss/kinde-android-generator , we're not expecting people to raise P.Rs to this repo.